src/stats/stats.h

```diff
@@ -37,6 +37,51 @@ enum StatsMetricFlags {
   STATS_METRIC_ROCKSDB_SEEK,      // Number of calls of seek in rocksdb
   STATS_METRIC_ROCKSDB_NEXT,      // Number of calls of next in rocksdb
   STATS_METRIC_ROCKSDB_PREV,      // Number of calls of prev in rocksdb
+  STATS_METRIC_BLOCK_CACHE_MISS,
+  STATS_METRIC_BLOCK_CACHE_HIT,
+  STATS_METRIC_BLOCK_CACHE_ADD,
+  STATS_METRIC_BLOCK_CACHE_BYTES_READ,
+  STATS_METRIC_BLOCK_CACHE_BYTES_WRITE,
+  STATS_METRIC_BLOOM_FILTER_USEFUL,
+  STATS_METRIC_BLOOM_FILTER_FULL_POSITIVE,
+  STATS_METRIC_BLOOM_FILTER_FULL_TRUE_POSITIVE,
+  STATS_METRIC_PERSISTENT_CACHE_HIT,
+  STATS_METRIC_PERSISTENT_CACHE_MISS,
```

Comment:
We have no persistent cache now, so it makes no sense to add those two.

Reply:
done